src/app/components/FrostedGlassPromo/index.jsx

```diff
@@ -15,8 +15,40 @@ import ImageWithPlaceholder from '../../containers/ImageWithPlaceholder';
 
 import withData from './withData';
 
-const Body = styled.div`
+const Wrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  text-decoration: none;
+  &:hover {
+    a {
+      text-decoration: underline;
+    }
+  }
+`;
+
+// This is an unfocusable element, hidden from screenreaders
+// This is to allow mouse users to click anywhere on the promo
+// But for keyboard and AT users, and scenarios where there is no CSS,
+// only the link inside the H3 should be interactive
```

Comment:
So this component now uses two `a` tags:

- this `a`, which is unfocusable, aria-hidden and covers the entire promo
- an "inner" `a` which just wraps the promo title, for AT users

Normally, we'd do this using a single `a` element - we'd wrap the title in an `a` and then style that `a` with a `:before` element that is absolutely positioned to fill its nearest relative ancestor, which would be the entire promo.

Wasn't able to make that work here though - I believe to implement the design, the bottom half of the promo where this link would be has to use a relative positioned wrapper.  This means an absolutely positioned `:before` inside that wrapper will only fill the bottom half of the promo, not the entire thing.

Reply:
Thought I'd see if there's a way to remove the extra anchor tag. With a little rejiggle of elements and styles I think it's possible https://github.com/bbc/simorgh/pull/9712 I hope it helps

Reply:
cheers for having a look - this doesn't seem to be the same design though when I pull the branch - the image behind the text is gone 😢 

This branch:
![Screen Shot 2021-12-01 at 10 37 07](https://user-images.githubusercontent.com/8601460/144219602-efd67b4e-02da-4c93-8ca7-c9a698461ba6.png)

That branch:
![Screen Shot 2021-12-01 at 10 36 28](https://user-images.githubusercontent.com/8601460/144219681-55b062dd-37e5-4029-a68c-abe87bda9e28.png)

Reply:
Thanks @jroebu14

Reply:
that's really strange, I don't get that. I see the image behind on Chrome and Firefox

Reply:
ah interesting - it looks different on storybook vs the page:

![Screen Shot 2021-12-01 at 15 55 51](https://user-images.githubusercontent.com/8601460/144268225-ba94d20c-71e2-41d3-9d64-294f3d431bdd.png)

Probably some z-index stuff happening on the page that's breaking it.  That is solvable, but it does look different still - that branch is making the image bigger, where the design was trying to get a "mirrored" effect:

before:
![Screen Shot 2021-12-01 at 15 59 14](https://user-images.githubusercontent.com/8601460/144268719-4e21838d-7ae1-43ae-871a-856d97f890ea.png)

after:
![Screen Shot 2021-12-01 at 15 59 24](https://user-images.githubusercontent.com/8601460/144268737-8beb090b-ef80-4502-9d2d-79951a202488.png)

 Not a huge deal and barely noticeable given how heavy the blur is - not entirely sure the second link is a big deal either though?  This technique is already being used on the news homepage
![Screen Shot 2021-12-01 at 16 04 06](https://user-images.githubusercontent.com/8601460/144269288-1cb002ff-2df9-4906-9fe2-0f81f864e669.png)

Reply:
A second link is less than optimal, this will been seen when CSS doesn't load for example. Also note the news example has content. If we can achieve this without 2 links, then we should please.

Reply:
@ryanmccombe May be worth asking UX if the slight design change using @jroebu14's approach is acceptable? Looks like a good way to do it if it avoids multiple anchor tags. Not sure of the SEO implications myself, so worthwhile checking the alternative approach.

Reply:
👍 UX said their original design was preferred if there was no technical reason to change it - was unable to find one, and we'll have more opportunities before this goes live

Reply:
Checked and it looks fine when CSS is disabled - only one link shown
There are no obvious SEO implications and the design has been agreed with Pete. 

Thanks Ryan.